exp/etcdrestore/controllers/etcdmachinesnapshot_controller.go

```diff
@@ -80,6 +80,11 @@ func (r *EtcdMachineSnapshotReconciler) Reconcile(ctx context.Context, req ctrl.
 		return ctrl.Result{}, err
 	}
 
+	if !etcdMachineSnapshot.Spec.Manual {
+		log.V(5).Info("Skipping snapshot creation for non-manual EtcdMachineSnapshot")
+		return ctrl.Result{}, nil
+	}
```

Comment:
```suggestion
	if etcdMachineSnapshot.Spec.Manual {
		log.V(5).Info("Skipping snapshot creation for manual EtcdMachineSnapshot")
		return ctrl.Result{}, nil
	}
```

Reply:
The `etcdmachinesnapshot_controller` has to trigger the creation of a snapshot using system-agent only if the EtcdMachineSnaphot was created by the user, which is a manual way. In any other case it means that the snapshot was created automatically with the sync controller.

Reply:
In this case the addition of manual field requires users to always specify it. This should be inversed IMO.

Reply:
I agree, we need a more UX friendly way. I'll open an issue for that.

Reply:
https://github.com/rancher/turtles/issues/728